### ISSUE TITLE
WindowsError: [Error 5]

### ISSUE THREAD

**Issue by RudeStef** [platformio/platformio-atom-ide#1191]
Virtualenv Create: Traceback (most recent call last):
  File "C:\Users\Rude\.platformio\.cache\tmp-8228dyppi1cT7457\virtualenv-14.0.6\virtualenv.py", line 2304, in <module>
    main()
  File "C:\Users\Rude\.platformio\.cache\tmp-8228dyppi1cT7457\virtualenv-14.0.6\virtualenv.py", line 705, in main
    symlink=options.symlink)
  File "C:\Users\Rude\.platformio\.cache\tmp-8228dyppi1cT7457\virtualenv-14.0.6\virtualenv.py", line 904, in create_environment
    site_packages=site_packages, clear=clear, symlink=symlink))
  File "C:\Users\Rude\.platformio\.cache\tmp-8228dyppi1cT7457\virtualenv-14.0.6\virtualenv.py", line 1079, in install_python
    mkdir(lib_dir)
  File "C:\Users\Rude\.platformio\.cache\tmp-8228dyppi1cT7457\virtualenv-14.0.6\virtualenv.py", line 317, in mkdir
    os.makedirs(path)
  File "C:\Users\Rude\.platformio\python27\lib\os.py", line 157, in makedirs
    mkdir(name, mode)
WindowsError: [Error 5] Acc�s refus: 'C:\\Users\\Rude\\.platformio\\penv\\Lib'

**Comment by ivankravets** [comment]
Please provide an output from system Terminal `cmd`
```
echo %PATH%
echo %PYTHONPATH%
python --version
pip --version
pip list
```

**Comment by ivankravets** [comment]
Please reopen if you still need help

**Comment by ivankravets** [comment]
Fixed in `platformio-ide` 2.1.5. Please upgrade.
